oauth2/org.wso2.dpop.proof.generator/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Add the license header

Reply:
fixed with https://github.com/wso2/samples-is/pull/486/commits/340ccf50cbf8a8c486e9b43d2438f001348050a1